Dockerfile

```diff
@@ -120,6 +120,13 @@ ENV CASSANDRA_CLUSTER=false \
 # Configure search nodes. Empty string falls back to search being done on the nodes themselves.
 ENV PEGA_SEARCH_URL=
 
+# Configure hazelcast. By default, hazelcast runs in embedded mode.
+Env CLIENT_MODE=false \
```

Comment:
Can we change the name of this env var to something that makes it clear it is related to Hazelcast?  e.g. HZ_CLIENT_MODE?

Reply:
Sure.